app/models/grda_warehouse/hud/client.rb

```diff
@@ -869,6 +869,10 @@ def inactivate_in_cas
       update(sync_with_cas: false)
     end
 
+    def self.hide_last_seen_info?
+      ENV.fetch('CLIENT_HIDE_LAST_SEEN_INFO', false)
+    end
```

Comment:
Sorry this wasn't clear, this should not be in ENV, but should come from `GrdaWarehouse::Config`

Reply:
It will need a migration, addition to the form, and to the permitted params for the admin/configs/controller

Reply:
No problem. I'll fix it tomorrow.

Reply:
✅